docs/dxp/latest/en/site-building/developer-guide/reference/fragments/fragment-specific-tags-reference.md

```diff
@@ -165,7 +165,7 @@ For Liferay Portal 7.3 GA2 and below, use this syntax:
 
 ## Including Widgets Within A Fragment
 
-To include a widget, you must know its registered name. For example, the Site Navigation Menu portlet is registered as `nav`. Each registered portlet has an `lfr-widget-[name]` tag that's used to embed it. For example: the Navigation Menu tag is `<lfr-widget-nav />`. You could embed it in a block like this:
+To include a widget, you must know its registered name. For example, the Menu Display Widget is registered as `nav`. Each registered portlet has an `lfr-widget-[name]` tag that's used to embed it. For example: the Menu Display tag is `<lfr-widget-nav />`. You could embed it in a block like this:
```

Comment:
Hi @Fabiomorais87, sometimes you use"Menu Display Widget" and other times "Menu Display" adding "widget" word in lowercase if needed. I think it should be consistent and use always the name "Menu Display". Thanks!